php7.4-cli/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM php:7.4-cli-alpine
+
+WORKDIR /opt
+
+# Install apk packages we want
+RUN apk add -Uuv \
+    git bash supervisor freetype-dev libjpeg-turbo-dev libzip-dev \
+    libpng-dev postgresql-dev  \
+    && rm -rf /var/cache/apk/*
+
+# Install wait-for-it
+RUN curl https://raw.githubusercontent.com/vishnubob/wait-for-it/master/wait-for-it.sh > /opt/wait-for-it.sh \
+    && chmod +x /opt/wait-for-it.sh \
+    && ln -s /opt/wait-for-it.sh /usr/bin/wait-for-it
+
+# Download and install composer
+ARG COMPOSER_COMMIT_HASH=da7be05fa1c9f68b9609726451d1aaac7dd832cb
+RUN wget https://raw.githubusercontent.com/composer/getcomposer.org/${COMPOSER_COMMIT_HASH}/web/installer -O - -q | php -- --quiet
+RUN chmod +x /opt/composer.phar \
+    && ln -s /opt/composer.phar /usr/bin/composer
+
+# Install PHP extensions
+ENV PHPREDIS_VERSION 5.3.2
```

Comment:
4.2.0 was not compiling on the 8.0 image so I decided to upgrade to latest for 7.4 and 8.0. The only breaking change from phpredis 4 -> 5 was dropping support for PHP5